src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -380,6 +383,27 @@ protected int doWork() {
         log.info(failedAlleleCheck, " variants lifted over but had mismatching reference alleles after lift over.");
         log.info(pct, " of variants were not successfully lifted over and written to the output.");
 
+        final Set<String> contigUnion = new TreeSet<>();
+        contigUnion.addAll(liftedBySourceContig.keySet());
+        contigUnion.addAll(rejectsByContig.keySet());
+
+        log.info("liftover success by source contig:");
+        for (String contig : contigUnion){
```

Comment:
space between ) and {

(also at several other places)

Reply:
updated 4 instances